be/src/exprs/vectorized/json_functions.cpp

```diff
@@ -425,6 +268,67 @@ ColumnPtr JsonFunctions::json_string(FunctionContext* context, const Columns& co
     return result.build(ColumnHelper::is_all_const(columns));
 }
 
+ColumnPtr JsonFunctions::json_int(FunctionContext* context, const Columns& columns) {
+    ColumnViewer<TYPE_JSON> viewer(columns[0]);
+    ColumnBuilder<TYPE_INT> result(columns[0]->size());
+
+    for (int row = 0; row < columns[0]->size(); row++) {
+        if (viewer.is_null(row)) {
+            result.append_null();
+        } else {
+            JsonValue* json = viewer.value(row);
+            auto json_int = json->get_int();
+            if (!json_int.ok()) {
+                result.append_null();
+            } else {
+                result.append(std::move(json_int.value()));
+            }
+        }
+    }
+    return result.build(ColumnHelper::is_all_const(columns));
+}
+
+ColumnPtr JsonFunctions::json_double(FunctionContext* context, const Columns& columns) {
+    ColumnViewer<TYPE_JSON> viewer(columns[0]);
+    ColumnBuilder<TYPE_DOUBLE> result(columns[0]->size());
+
+    for (int row = 0; row < columns[0]->size(); row++) {
+        if (viewer.is_null(row)) {
+            result.append_null();
+        } else {
+            JsonValue* json = viewer.value(row);
+            auto json_d = json->get_double();
+            if (!json_d.ok()) {
+                result.append_null();
+            } else {
+                result.append(std::move(json_d.value()));
+            }
+        }
+    }
+    return result.build(ColumnHelper::is_all_const(columns));
+}
+
+ColumnPtr JsonFunctions::string_json(FunctionContext* context, const Columns& columns) {
```

Comment:
Why the other function name json_int/json_double, this function is renamed string_json other than json_string?